.storybook/main.ts

```diff
@@ -14,21 +11,27 @@ module.exports = {
         prop.parent ? !/node_modules/.test(prop.parent.fileName) : true,
     },
   },
-  stories: ['../src/**/ui/FormField/*.stories.@(ts|js|tsx|mdx)'],
+  stories: [
```

Comment:
Should this contain wildcard for supporting all stories over Badge now (not sure why this had only FormField in the past)?

Reply:
It is wildcard matching in the latest commit. 
```
stories: ['../src/**/*.@(stories.tsx|mdx)'],
```
Probably because you were checking the old commit where it has not been changed.